text/0258-env-context-baggage-carriers.md

```diff
@@ -0,0 +1,359 @@
+# Environment Variables as Context and Baggage Carriers
+
+Add requirements for inter-process context and baggage propagation using
+environment variables as the carrier.
+
+This OTEP is based on discussion in [opentelemetry-specification #740](https://github.com/open-telemetry/opentelemetry-specification/issues/740#issue-665588273), builds on the [Context Propagators
+API](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/context/api-propagators.md)
+and follows [OTEP 0205: Context propagation requirements for messaging semantic conventions](https://github.com/open-telemetry/oteps/blob/main/text/trace/0205-messaging-semantic-conventions-context-propagation.md)
+as a guide.
+
+## Motivation
+
+There are existing requirements for context propagation over HTTP,
+messaging systems, and others, but there are no requirements for
```

Comment:
> there are no requirements

in accurate - there are both requirements and implementations, what's lacking is a standard/specification.

Reply:
Fair point on the wording here, I'll adjust.